src/app/modules/main/chat_section/module.nim

```diff
@@ -7,6 +7,11 @@ import model as chats_model
 import item as chat_item
 import ../../shared_models/user_item as user_item
 import ../../shared_models/user_model as user_model
+import ../communities/models/token_permissions_model
+import ../communities/models/token_permission_item
+import ../communities/models/token_criteria_item
+import ../communities/models/token_criteria_model
+import ../communities/models/token_list_item
```

Comment:
Maybe those models and items should be put in `shared_models`?

Reply:
Can move them there for sure!
Started having them here because I believe we're only using them in the context of communities (and chat sections that **are** communities)